### PR TITLE
[bn] Localize serverless.md

### PR DESCRIPTION
**Pull request by mahadihassanriyadh**
Signed-off-by: Md. Mahadi Hassan Riyadh [mahadihassanriyadh1@gmail.com](mailto:mahadihassanriyadh1@gmail.com)
Localizes content/bn/serverless.md to Bengali
This is an initial PR as may need some improvements!
Tracking issue: https://github.com/cncf/glossary/issues/410

### CODE REVIEW

**Comment by mahadihassanriyadh** [comment]
@mitul3737 Are there any issues with this pr?

File: content/bn/serverless.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+Title: সার্ভারহীন (Serverless)
+Status: Completed
+Category: প্রযুক্তি
+tags: ["architecture", "", ""]
+---
+
+## এটা কি
+
+সার্ভারলেস হল একটি ক্লাউড নেটিভ ডেভেলপমেন্ট মডেল যা ডেভেলপারদের সার্ভার পরিচালনা না করেই অ্যাপ্লিকেশন তৈরি এবং চালানোর অনুমতি দেয়। এখনও সার্ভারহীনে সার্ভার আছে, কিন্তু তারা অ্যাপ ডেভেলপমেন্ট থেকে [বিমূর্ত](/abstraction/)(abstracted) দূরে। একজন ক্লাউড প্রদানকারী (cloud provider) সার্ভার পরিকাঠামোর প্রভিশনিং, রক্ষণাবেক্ষণ এবং [স্কেলিং](/scalability/) এর রুটিন কাজ পরিচালনা করে। ডেভেলপাররা তাদের কোড কেবল স্থাপনের (deployment) জন্য [কন্টেইনার](/container/) এ প্যাকেজ করতে পারে। একবার স্থাপন করা হয়ে গেলে, সার্ভারহীন অ্যাপগুলি চাহিদার (demand) প্রতি সাড়া দেয় এবং প্রয়োজন অনুসারে স্বয়ংক্রিয়ভাবে সার্ভারের ক্ষমতা বাড়ায় এবং কমায়। পাবলিক ক্লাউড প্রদানকারীদের সার্ভারহীন অফারগুলি সাধারণত একটি ইভেন্ট-চালিত (event-driven) এক্সিকিউশন মডেলের মাধ্যমে চাহিদা অনুযায়ী পরিমাপ করা হয়। ফলস্বরূপ, যখন একটি সার্ভারহীন ফাংশন নিষ্ক্রিয় বসে থাকে, তখন এটির জন্য কিছু খরচ হয় না।
```
Comment (mitul3737) [inline content/bn/serverless.md:10]:
```suggestion
সার্ভারলেস হল একটি ক্লাউড নেটিভ ডেভেলপমেন্ট মডেল যা ডেভেলপারদের সার্ভার পরিচালনা না করেই অ্যাপ্লিকেশন তৈরি এবং চালানোর অনুমতি দেয়। এখনও সার্ভারহীনে সার্ভার আছে, কিন্তু তারা অ্যাপ ডেভেলপমেন্ট থেকে [বিমূর্ত](/bn/abstraction/)(abstracted) দূরে। একজন ক্লাউড প্রদানকারী (cloud provider) সার্ভার পরিকাঠামোর প্রভিশনিং, রক্ষণাবেক্ষণ এবং [স্কেলিং](/bn/scalability/) এর রুটিন কাজ পরিচালনা করে। ডেভেলপাররা তাদের কোড কেবল স্থাপনের (deployment) জন্য [কন্টেইনার](/bn/container/) এ প্যাকেজ করতে পারে। একবার স্থাপন করা হয়ে গেলে, সার্ভারহীন অ্যাপগুলি চাহিদার (demand) প্রতি সাড়া দেয় এবং প্রয়োজন অনুসারে স্বয়ংক্রিয়ভাবে সার্ভারের ক্ষমতা বাড়ায় এবং কমায়। পাবলিক ক্লাউড প্রদানকারীদের সার্ভারহীন অফারগুলি সাধারণত একটি ইভেন্ট-চালিত (event-driven) এক্সিকিউশন মডেলের মাধ্যমে চাহিদা অনুযায়ী পরিমাপ করা হয়। ফলস্বরূপ, যখন একটি সার্ভারহীন ফাংশন নিষ্ক্রিয় বসে থাকে, তখন এটির জন্য কিছু খরচ হয় না।
```

File: content/bn/serverless.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+## এটা যেসব সমস্যাতে দৃষ্টিপাত করে
+একটি স্ট্যান্ডার্ড [Infrastructure-as-a-service (IaaS)](/infrastructure-as-a-service/) [ক্লাউড কম্পিউটিং](/cloud-computing/) মডেলের অধীনে,
```
Comment (mitul3737) [inline content/bn/serverless.md:13]:
```suggestion
একটি স্ট্যান্ডার্ড [Infrastructure-as-a-service (IaaS)](/bn/infrastructure-as-a-service/) [ক্লাউড কম্পিউটিং](/bn/cloud-computing/) মডেলের অধীনে,
```

**Review by mitul3737 — CHANGES_REQUESTED** [review]

File: content/bn/serverless.md (hunk 1 above)
Comment (mahadihassanriyadh) [reply content/bn/serverless.md]: Done ✅

File: content/bn/serverless.md (hunk 2 above)
Comment (mahadihassanriyadh) [reply content/bn/serverless.md]: Done ✅

**Comment by mahadihassanriyadh** [comment]
@mitul3737 I have resolved the issues.

**Review by mitul3737 — APPROVED** [review]

**Review by Imtiaz1234 — APPROVED** [review]

**Pull request merged**